texastribune/donations
# Border coverage 2024

#### What's this PR do?
- adds standard yellow box for a fund raiser promo
- add a new social share graphic
- introduces a new version of the thermometer that just outputs a value

| Before | After |
|--------|--------|
| <img width="1111" alt="Screenshot 2024-04-12 at 2 03 37 PM" src="https://github.com/texastribune/donations/assets/2974713/5f4496f8-3647-40d7-bd6c-c9d8251b7dd0"> | <img width="1105" alt="Screenshot 2024-04-12 at 2 03 21 PM" src="https://github.com/texastribune/donations/assets/2974713/97551e25-6b8b-4393-9771-755db1827285">
 |


#### Why are we doing this? How does it help us?

Helps us continue to cover all the news from the US/Mexico border


#### How should this be manually tested?

`make`
Confirm that the new message appears and the new social graphic is referenced

#### How should this change be communicated to end users?

Emily knows I'm going to deploy this first thing Monday

#### Are there any smells or added technical debt to note?

Yes, I added the new "minimal" thermometer in a way that isn't very DRY. If we keep wanting one like this, we should figure out a cleaner approach.

#### What are the relevant tickets?

https://airtable.com/appyo1zuQd8f4hBVx/tbloNZu8GkM52NKFR/viwb2L2CHc4ZWqkC5/recKyxS2nxqcvK7p6?blocks=hide

#### TODOs / next steps:

* [ ] *Deploy 4/15*
